Description: Include the words (Top XXX) in the ExtraSection headings.
Author: Patrick Collins <pcollins1@users.sourceforge.net>
Bug: https://sourceforge.net/p/awstats/patches/135

---
 wwwroot/cgi-bin/awstats.pl |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/wwwroot/cgi-bin/awstats.pl
+++ b/wwwroot/cgi-bin/awstats.pl
@@ -11757,7 +11757,7 @@ sub HTMLShowExtraSections{
 			if ($Debug) { debug( "ExtraName$extranum", 2 ); }
 			print "$Center<a name=\"extra$extranum\">&nbsp;</a><br />";
 			my $title = $ExtraName[$extranum];
-			&tab_head( "$title", 19, 0, "extra$extranum" );
+			&tab_head( "$title ($Message[77] $MaxNbOfExtra[$extranum])", 19, 0, "extra$extranum");
 			print "<tr bgcolor=\"#$color_TableBGRowTitle\">";
 			print "<th>" . $ExtraFirstColumnTitle[$extranum] . "</th>";
 
@@ -16845,7 +16845,7 @@ sub HTMLMainExtra{
 	if ($Debug) { debug( "ExtraName$extranum", 2 ); }
 	print "$Center<a name=\"extra$extranum\">&nbsp;</a><br />";
 	my $title = $ExtraName[$extranum];
-	&tab_head( "$title", 19, 0, "extra$extranum" );
+	&tab_head( "$title ($Message[77] $MaxNbOfExtra[$extranum])", 19, 0, "extra$extranum");
 	print "<tr bgcolor=\"#$color_TableBGRowTitle\">";
 	print "<th>" . $ExtraFirstColumnTitle[$extranum];
 	print "&nbsp; - &nbsp; <a href=\""
